src/riak_kv_crdt.erl

```diff
@@ -411,10 +416,27 @@ to_record(?SET_TYPE, Val) ->
     ?SET_TYPE(Val).
 
 %% @doc Check cluster capability for crdt support
-%% @TODO what does this mean for Maps?
 supported(Mod) ->
     lists:member(Mod, riak_core_capability:get({riak_kv, crdt}, [])).
 
+%% @private get the binary version for a crdt mod, default to `1' for
+%% pre-versioned.
+-spec crdt_version(module()) -> pos_integer().
+crdt_version(Mod) ->
+    %% due to the riak-2.0.4 disaster where mixed format maps were
+    %% written to disk (see riak#667 for more) override the cluster
+    %% negotiated capability with an env var, this is to ensure that
+    %% in a multi-cluster environment, Epoch 1 binary format is used until
+    %% all clusters are Epoch 2 capable.
+    case app_helper:get_env(riak_kv, mdc_crdt_epoch) of
+        1 ->
+            proplists:get_value(Mod, ?E1_DATATYPE_VERSIONS, 1);
+        _ ->
+            %% use any term except true to unset app env. Default to
```

Comment:
With the name and semantic changes, this comment is now minorly misleading. Any value other than `1` triggers this clause.